agave_pyclient/agave_pyclient/commandbuffer.py

```diff
@@ -78,6 +78,8 @@
     "SET_FLIP_AXIS": [46, "I32", "I32", "I32"],
     "SET_INTERPOLATION": [47, "I32"],
     "SET_CLIP_PLANE": [48, "F32", "F32", "F32", "F32"],
+    # channel index, colormap name, array of [stop, r, g, b, a]
+    "SET_COLOR_RAMP": [49, "I32", "S", "F32A"],
```

Comment:
defines the binary footprint of the arguments for this command, details in command.h and command.cpp